app/Lesson.php

```diff
@@ -14,4 +14,8 @@ public function tutor()
   {
     return $this->belongsTo('App\Tutor', 'tutor_id', 'user_id');
   }
+
+  public function subject() {
+    return $this->belongsTo('App\Subjects', 'subject_id', 'id');
```

Comment:
`return $this->belongsTo('App\Subject');`
Check by using:
`php artisan tinker`
`Lesson::find(1)->subject`